hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.tsx

```diff
@@ -34,6 +37,7 @@ const CartSummary = () => {
     const isOutsideSignOutPeriod =
         currentDateTime < hardwareSignOutStartDate ||
         currentDateTime > hardwareSignOutEndDate;
+    const isTestUser = user?.groups.find((group) => group.name === hssTestUserGroup);
```

Comment:
Is it a good idea to have a selector just to get the Test User in the userSlice? It's used pretty often in a few files, and I think it would look cleaner.

Reply:
will do